lib/kernel/doc/src/erl_epmd.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE erlref SYSTEM "erlref.dtd">
+
+<erlref>
+  <header>
+    <copyright>
+      <year>2018</year><year>2018</year>
+      <holder>Ericsson AB. All Rights Reserved.</holder>
+    </copyright>
+    <legalnotice>
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+
+    </legalnotice>
+
+    <title>erl_epmd</title>
+    <prepared>Timmo Verlaan</prepared>
+    <docno>1</docno>
+    <date>2018-02-19</date>
+    <rev>A</rev>
+  </header>
+  <module>erl_epmd</module>
+  <modulesummary>
+    Erlang interface towards <seealso marker="erts:epmd"><c>epmd</c></seealso>
```

Comment:
`xmllint` fails here since `modulesummary` may only contain plain text. The rest looks great though!